slsa_for_models/README.md

```diff
@@ -0,0 +1,37 @@
+# SLSA for Models
+
+To protect the supply chain of traditional software against tampering (like in
+the [Solarwinds attack][solarwinds]), we can generate [SLSA][slsa] provenance,
+for example by using the [SLSA L3 GitHub generator][slsa-generator].
+
+This projects shows how we can use the same generator for training models via
+GitHub Actions. While most of the ML models are too expensive to train in such a
+fashion, this is a proof of concept to prove that _the same traditional software
+supply chain protections can be applied to ML_. Future work will involve
+covering training ML models that require access to accelerators (i.e., GPUs,
+TPUs) or that require multiple hours for training.
+
+When users download a given version of a model they can also check its
+provenance by using [the SLSA verifier][slsa-verifier] repository. This can be
+done automatically: for example the model serving pipeline could validate
+provenance for all new models before serving them. The verification can also be
+done manually, on demand.
+
+As an additional benefit, having a provenance for a model allows users to react
```

Comment:
Nit: change "having a provenance" to "having provenance" (or "having the provenance").

Reply:
Thank you, changed!